zebedee-reader/src/main/java/com/github/onsdigital/zebedee/content/page/home/HomeContentItem.java

```diff
@@ -0,0 +1,38 @@
+package com.github.onsdigital.zebedee.content.page.home;
+
+import com.github.onsdigital.zebedee.content.partial.Image;
+
+import java.net.URI;
+
+public class HomeContentItem {
+
+    private String title;
+    private URI uri;
+    private String description;
+    private Image image;
```

Comment:
Add 2 constructors 1 which is empty, the 2nd taking these fields and setting them on the object. Makes it nicer for someone else to use.